packages/adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shapeshiftoss/metamask-snaps-adapter",
-  "version": "1.0.10",
+  "version": "1.0.11",
```

Comment:
This repo doesn't autopublish, will publish once stamped

Reply:
Published https://www.npmjs.com/package/@shapeshiftoss/metamask-snaps-adapter/v/1.0.11?activeTab=code